messaging/src/main/java/org/axonframework/eventhandling/pooled/WorkPackage.java

```diff
@@ -520,6 +523,7 @@ static class Builder {
         private long claimExtensionThreshold = 5000;
         private Consumer<UnaryOperator<TrackerStatus>> segmentStatusUpdater;
         private Clock clock = GenericEventMessage.clock;
+        private Runnable clearCacheFunction;
```

Comment:
If we rename the operation on the `EventHandlerInvoker`, I anticipate we rename the field name too.
Perhaps something like `shutdownAction` (as present in the `Coordinator`) or `abortAction` would make sense.

Reply:
It's now done from the `Coordinator` on releasing the token. So no changes to the `Workpackage` are needed anymore.